Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:3.11
+
+RUN apk add curl
+
+RUN apk add python3
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
+
+COPY . .
```

Comment:
This currently copies everything in your checkout of this repo into the docker image - including any secrets set in the .env file. While that's not a security problem right now, it would be if we wanted to share the docker image, so we should add a `.dockerignore` file to prevent those being copied into our images (and we can take advantage of that to cut out any large/irrelevant files too, like the `.vagrant` folder)

Reply:
ty, done